modules/developer_manual/pages/_partials/app/advanced/security/idp_setup/node-oidc-provider-config.js

```diff
@@ -0,0 +1,9 @@
+module.exports.clients = [
```

Comment:
to be removed

Reply:
The entire file?

Reply:
Does the entire section need to be removed?

Reply:
yes - this holds values of the nodejs idp - which is for dev purpose - not reasonable for the admin docs

Reply:
Though it's already in the developer docs, not the admin docs.

Reply:
let's just kill this please - it is only me at the end who used it for testing frontchannel logout ....